atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/paging/RowGetter.java

```diff
@@ -94,4 +94,4 @@ private SlicePredicate getSlicePredicate(ColumnFetchMode fetchMode) {
         predicate.setSlice_range(slice);
         return predicate;
     }
-}
+}
```

Comment:
New line, or the file appears in the diff `git diff 0.60.1..HEAD --name-only | grep cassandra`.